sql/testdata/select_index

```diff
@@ -164,8 +158,9 @@ EXPLAIN (DEBUG) SELECT * FROM t@ab WHERE a >= 3 OR a > 3
 query ITTB
 EXPLAIN (DEBUG) SELECT * FROM t@ab WHERE a = 3 OR a = 5
 ----
-0 /t/ab/3/4 NULL true
-1 /t/ab/5/6 NULL true
+0 /t/ab/1/2 NULL false
+1 /t/ab/3/4 NULL true
+2 /t/ab/5/6 NULL true
```

Comment:
The prior code would transform `a = 3 OR b = 5` to `a >= 3 AND b <= 5`. Now we're keeping the disjunction expressions separate, but I still have to implement the code which will do multiple lookups into an index.